packages/cubejs-testing/src/db/materialize.ts

```diff
@@ -0,0 +1,27 @@
+import { GenericContainer } from 'testcontainers';
+
+import { DbRunnerAbstract, DBRunnerContainerOptions } from './db-runner.abstract';
+
+type MaterializeStartOptions = DBRunnerContainerOptions & {
+  version?: string,
+};
+
+export class MaterializeDBRunner extends DbRunnerAbstract {
+  public static startContainer(options: MaterializeStartOptions) {
+    const version = process.env.TEST_MZSQL_VERSION || options.version || 'v0.24.0';
```

Comment:
Please add a comment explaining how `v0.24.0` was chosen and why/how to update it.